### DISCUSSION TITLE
Grabar, tipo DVR

### DISCUSSION THREAD

**Discussion by javidial** [Paco8/plugin.video.movistarplus#12] [Ideas]
Ya que se puede ver el contenido en Kodi, se puede grabar o programar una grabacion desde kodi?

No se si es posible. Lo he intentado con el "IPTV Recorder", pero da error. Supongo que al tener DRM, no puede ser grabado, pero si kodi tiene widevine y se puede ver, que lo impide que el propio programa pueda capturarla sin recompresion.

Gracias.

**Comment by Paco8** [comment]
Eso ya sería cosa del inputstream.adaptative y no lo van a implementar.
https://github.com/xbmc/inputstream.adaptive/issues/238
